From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Fri, 18 Jul 2014 06:33:52 +0200
Subject: install: systemd-timesyncd.service is enabled by sysinit.target

systemd-timesyncd.service has a "WantedBy=sysinit.target" so the
initially generated link should match that.

(cherry picked from commit e9b11a8457293c553296e5d986a0bb7f86f275d5)
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 2b0c855..53f82f9 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -4346,7 +4346,7 @@ nodist_systemunit_DATA += \
 	units/systemd-timesyncd.service
 
 GENERAL_ALIASES += \
-	$(systemunitdir)/systemd-timesyncd.service $(pkgsysconfdir)/system/multi-user.target.wants/systemd-timesyncd.service
+	$(systemunitdir)/systemd-timesyncd.service $(pkgsysconfdir)/system/sysinit.target.wants/systemd-timesyncd.service
 
 EXTRA_DIST += \
 	units/systemd-timesyncd.service.in
